.github/workflows/go-test.yaml

```
@@ -70,6 +70,13 @@ jobs:
           set -ex
           kubectl rook-ceph rbd ls replicapool
 
+      - name: Subvolume command
```

```suggestion
      - name: Subvolume command
```

same command needs to be added below for other namespaces